TSC-Charter.md

```diff
@@ -56,14 +57,29 @@ one-fourth of the TSC membership shares an employer, then the situation
 must be immediately remedied by the resignation or removal of one or more
 TSC members affiliated with the over-represented employer(s).
 
-The TSC members shall consist of Maintainers from Core Projects as
-defined in the project lifecycle document and Section 7.
+The TSC membership shall include (but is not limited to) Maintainers from Core
+Projects as defined in the project lifecycle document and Section 7. The TSC
+may, at its discretion, invite new members to join the TSC who are not
+considered to be Maintainers from Core Projects.
+
+The TSC may, at its discretion, invite any number of non-voting observers to
+participate in TSC meetings.
 
 The TSC shall meet regularly using tools that enable participation by the
 community (e.g. weekly on a Google Hangout On Air, or through any other
 appropriate means selected by the TSC). The meeting shall be directed by
-the TSC Chairperson. Minutes or an appropriate recording shall be taken
-and made available to the community through accessible public postings.
+the TSC Chairperson. Responsibility for directing individual meetings may be
+delegated by the TSC Chairperson to any other TSC member. Minutes or an
+appropriate recording shall be taken and made available to the community
+through accessible public postings.
+
+In the case where an individual TSC member attends fewer than 20% of the
+regularly scheduled meetings held within a six month period, and does not
```

Comment:
Id' favor being a bit conservative in the number.  25% seems more reasonable, but I'm also ok with a secondary measure like "and must attend at least 1 meeting every 3 months) so that if people just stop participating at all it raises a flag.  I see the case were we have not seem somebody for 3 months as quite different than the case were we see that person only 1/4 meetings but they are there for at least one every month.